README.md

```diff
@@ -58,6 +58,7 @@ Sorted alphabetically
 | [:link:](https://customelements.io) | CustomElements | `1.1` | | [Source](https://github.com/customelements/www) |
 | [:link:](http://www.ea.com) | Electronic Arts | `1.8` | | |
 | [:link:](https://frontend.directory) | Frontend Directory | `1.6` | | |
+| [:link:](https://frontend.directory) | Festilo - A mobile local shopping app | `1.05` | | |
```

Comment:
It might be worth having some contributing guidelines with tips and tricks, such as typing `Polymer.version` in the console returns the version (I doubt many people know that)